src/main/scala/nl.knaw.dans.easy.bag2deposit/DepositPropertiesFactory.scala

```diff
@@ -82,7 +85,7 @@ case class DepositPropertiesFactory(configuration: Configuration, idType: IdType
         case _ =>
       }
       addProperty("dataverse.bag-id", "urn:uuid:" + bagInfo.uuid)
-      if (!configuration.dansDoiPrefixes.contains(doi.replaceAll("/.*", ""))) {
+      if (!dansDoiPrefixes.contains(doi.replaceAll("/.*", ""))) {
```

Comment:
this is still not enough. i guess `doi` also contains the 'http' part. we need to get to the prefix only